src/json.jl

```diff
@@ -5,7 +5,7 @@
 Read a GeoJSON string to a GeoInterface.jl compatible feature or geometry object.
 """
 function read(source)
-    object = JSON3.read(source)
+    object = JSON3.read(source; numbertype = Float64)
```

Comment:
Should we allow passing the `numbertype` keyword through `read`?

Reply:
I think not using this setting is a real footgun, not sure we should offer that option. I doubt it's ever useful in practice.

Reply:
Yeah, makes sense. I just saw the tests had used `Int` previously and wondered if we should facilitate `numvertype=Int` in case anyone else was using `Int`. But I guess that's not really a real scenario.